src/users_db.rs

```diff
@@ -389,14 +385,22 @@ impl UsersDb {
         let mut users = Vec::new();
         for result_row in rows {
             let row = try!(result_row);
-            users.push(User {
+            let u = User {
```

Comment:
nit: use `user` instead of `u`. avoid using single letter (non descriptive) variable names.